### PR TITLE
Added Answers to a couple of questions

### PR DESCRIPTION
**Pull request by SajjadPSavoji**
Questions that are affected by this merge are:
Q4, Q6, Q7, Q15, Q16, Q19, Q57.

All modifications have appropriate online references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,9 +368,10 @@ The biggest difference between the the output of UMAP when compared with t-SNE i
 It generates a pseudo random number based on the seed and there are some famous algorithm, please see below link for further information on this.
 [[src]](https://en.wikipedia.org/wiki/Linear_congruential_generator)
 
+#### 64) What is the difference between Bayesian vs frequentist statistics? [[src]](https://www.kdnuggets.com/2022/10/nlp-interview-questions.html)
+
+Frequentist statistics is a framework that focuses on estimating population parameters using sample statistics, and providing point estimates and confidence intervals.
+
+Bayesian statistics, on the other hand, is a framework that uses prior knowledge and information to update beliefs about a parameter or hypothesis, and provides probability distributions for parameters.
 
-## Contributions
```
Comment (andrewekhalel) [inline README.md:367]: Why did you remove the contributions sections?

File: NLP/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# NLP Interview Questions
```
Comment (andrewekhalel) [inline NLP/README.md:1]: Thanks @SajjadPSavoji! A great addition to the repo. 
Looking forward to more contributions on the NLP part!

**Review by andrewekhalel — COMMENTED** [review]
Thanks @SajjadPSavoji for contributing. I can merge your PR if you fix the ["Contribution" section](https://github.com/andrewekhalel/MLQuestions/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L336)

**Review by SajjadPSavoji — COMMENTED** [review]
Will add the contribution part, it got deleted by mistake

File: NLP/README.md (hunk 2 above)
Comment (SajjadPSavoji) [reply NLP/README.md:1]: Thanks, I'll try to add more content later on

**Comment by andrewekhalel** [comment]
Awesome work! Thanks again
